packages/web/src/common/store/cache/sagas.js

```diff
@@ -98,15 +99,21 @@ export function* retrieve({
     call(getEntriesTimestamp, uniqueIds)
   ])
 
+  const confirmer = yield select(getConfirmCalls)
   // Figure out which IDs we need to retrive from source
   const idsToFetch = []
   uniqueIds.forEach((id) => {
-    if (
+    let isConfirming = false
+    if (cachedEntries?.[id]?.[idField]) {
```

Comment:
Awesome adds. Isn't the cache add step check if confiming, and if so don't call add_successful?

Reply:
You're right, we already had code to do this. It seems like pull to refresh explicitly passes through an arg to blow away the current entry in the cache. Maybe we can huddle abt this, not quite sure what the reasoning for that was

Reply:
Ok I removed the code I added here and also dropped the arg in pull to refresh and I see the same behavior as with my fix. Mine might be better b/c we lose a network hop, but it's definitely a riskier change so will merge without.

Reply:
Whatever you think is best, this can be something we discuss during irl too